### PR TITLE
[API] Added Payment Controller and dtos

### PR DESCRIPTION
**Pull request by outref**

### CODE REVIEW

File: src/main/java/com/example/carsharingservice/repository/PaymentRepository.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package com.example.carsharingservice.repository;
 
 import com.example.carsharingservice.model.Payment;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/repository/PaymentRepository.java:8]: double empty line

File: src/main/java/com/example/carsharingservice/repository/PaymentRepository.java [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 
 import com.example.carsharingservice.model.Payment;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+
 @Repository
 public interface PaymentRepository extends JpaRepository<Payment, Long> {
+    List<Payment> findAllByRental_User_Id(Long userId);
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/repository/PaymentRepository.java:11]: very bad naming for method (we can not use _ )

File: src/main/java/com/example/carsharingservice/service/PaymentService.java [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 import com.example.carsharingservice.model.Payment;
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/service/PaymentService.java:4]: empty line

File: src/main/java/com/example/carsharingservice/controller/PaymentController.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/payments")
+public class PaymentController {
+    private final PaymentService paymentService;
+    private final PaymentMapper mapper;
+
+    @GetMapping
+    public List<PaymentResponseDto> getPaymentsByUser(@RequestParam Long userId) {
```
Comment (OlehSarapuk) [inline src/main/java/com/example/carsharingservice/controller/PaymentController.java:27]: it would be great to add pagination and swagger :)

**Review by OlehSarapuk — CHANGES_REQUESTED** [review]

**Review by OlehSarapuk — APPROVED** [review]

**Review by IvanZuravlev228 — APPROVED** [review]

**Pull request merged**